premarum-backend/PreEnrollmentMgmt.WebApi/Controllers/DTOS/CoursesTakenDTO.cs

```diff
@@ -0,0 +1,11 @@
+using PreEnrollmentMgmt.Core.Entities;
+
+namespace PreEnrollmentMgmt.WebApi.Controllers.DTOS;
+
+public class CoursesTakenDTO
+{
+    public int studentId { get; set; }
+    public int couseId { get; set; }
+    public int semesterId { get; set; }
+    public IEnumerable<CoursesTaken> coursesTaken { get; set; }
```

Comment:
Don't return the student Id. 
 CourseId and semesterId is enough. for the CoursesTakenDto.